debian/control

```diff
@@ -17,17 +18,19 @@ Build-Depends-Indep: python3-dev,
                      python3-apt,
                      python3-mock,
                      lsb-release
-Standards-Version: 3.9.3
+Standards-Version: 4.1.4
 Vcs-Git: https://github.com/mvo5/unattended-upgrades.git
 
 Package: unattended-upgrades
 Architecture: all
 Depends: ${shlibs:Depends},
          ${misc:Depends},
          debconf,
+         gir1.2-glib-2.0,
```

Comment:
Can we make those dependencies optional please? And handle them missing gracefully via import errors and warnings?

Reply:
The packages are present even in minimal lxc containers.

Reply:
It's fine, if it bothers me too much I can just make a PR to fix it - its just my instinct to avoid the extra 3 packages (and ~1MB) for a (admittedly nice) feature that is off by default (especially since python handles missing imports so nicely).

Reply:
I'm trying to avoid extra dependencies, too, unless they are present anyway practically everywhere.